book/src/running-validator/validator-reqs.md

```diff
@@ -0,0 +1,47 @@
+# Hardware and Software Requirements
+
+## Hardware
+
+See our [suggested hardware configurations here](https://solana-labs.github.io/book/testnet-participation.html#machine-requirements).
+
+* CPU Recommendations
+  * We recommend a CPU with the highest number of cores as possible. AMD Threadripper or Intel Server \(Xeon\) CPUs are fine. 
+  * We recommend AMD Threadripper as you get a larger number of cores for parallelization compared to Intel. 
+  * Threadripper also has a cost-per-core advantage and a greater number of PCIe lanes compared to the equivalent Intel part. PoH \(Proof of History\) is based on sha256 and Threadripper also supports sha256 hardware instructions.
+* SSD size and I/O style \(SATA vs NVMe/M.2\) for a validator
+  * Minimum example - Samsung 860 Evo 2TB
+  * Mid-range example - Samsung 860 Evo 4TB
+  * High-end example - Samsung 860 Evo 4TB
+* GPUs
+  * **Validator** nodes will be required to run with GPUs starting at Stage 1 of Tour de SOL. Without GPUs, a validator will not be able to catch up to the ledger once the network is launched. GPUs are NOT required for validators during Stage 0/Dry Runs of Tour de SOL.
```

Comment:
Let's remove the reference to TdS. Maybe instead, something like:
"While a CPU-only node may be able to keep up with the initial idling network, once transaction throughput increases, GPUs will be necessary"